### ISSUE TITLE
Staging Review finding: Links open in a new tab without warning

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#96652]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** My Education Benefits
**Product:** 22-5490 Form
**Feature:** (Fry/DEA) Updates

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility), @erinrwhite (IA)

### Description

There are multiple instances where a link in the workflow opens in a new tab without notifying the user that it's going to open in a new tab:
- [/benefit-selection](https://staging.va.gov/education/survivor-dependent-education-benefit-22-5490/benefit-selection), the "learn more" links
- [/contact-information](https://staging.va.gov/education/survivor-dependent-education-benefit-22-5490/contact-information) and [/contact-information/mailing-address](https://staging.va.gov/education/survivor-dependent-education-benefit-22-5490/contact-information/mailing-address), the "Go to your profile" links
- [/contact-information/contact-method](https://staging.va.gov/education/survivor-dependent-education-benefit-22-5490/contact-information/contact-method), the text notification terms and conditions and the privacy policy links

Opening in a new tab can be disorienting for a variety of users, but is especially disorienting for screen reader users as not all screen readers announce the change of context.

## Recommended action

Warn users inside the link text, eg. `<a href="..." target="_blank">Learn more about DEA education benefits (opens in a new tab)</a>`.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** [Link ](https://design.va.gov/components/language-toggle)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** [Links](https://design.va.gov/content-style-guide/links)
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
